### ISSUE TITLE
Priority changes not recognized when moving blank to a different position

### ISSUE THREAD

**Issue by Nlar** [GothenburgBitFactory/taskwarrior#2409]
Versions 2.6.0 and 2.5.1.  

**Observed:**
    tw
    [tw next]

    ID Age P Description       Urg 
    2 3s  L My Low Priority    1.8
    1 14s   My Blank Priority    0

Configuration Change
   ./tw show  | grep -A1 "uda.priority.values\|urgency.uda.priority"

    uda.priority.values              H,M,,L
    Default value                    H,M,L,
    --
    urgency.uda.priority.H.coefficient 6.0
    urgency.uda.priority.L.coefficient 1.8
    urgency.uda.priority.M.coefficient 3.9
    urgency.user.tag.next.coefficient  15.0

**Expected:**  Blank priority is higher than the low priority

    ID Age  P Description       Urg 
    1 3min   My Blank Priority  2.5
    2 3min L My Low Priority   1.8

I added:
    `urgency.uda.priority..coefficient=2.5 ` in my .taskrc file
for the expected.

I believe this is by design.  I can updated the wiki, but wanted to confirm as not a bug before creating documentation that causes further issues.  This method is listed explicitly to change this behaviour.  https://taskwarrior.org/docs/priority.html Section Custom Configuration.

**For completeness**
    ./tw diag    
    
    task 2.6.0    
       Platform: Linux    
    
    Compiler    
        Version: 9.3.0    
           Caps: +stdc +stdc_hosted +LP64 +c8 +i32 +l64 +vp64 +time_t64    
     Compliance: C++17    
    
    Build Features    
          Built: Feb  2 2021 09:25:20    
         Commit: cdbab698e    
          CMake: 3.17.4    
        libuuid: libuuid + uuid_unparse_lower    
      libgnutls: 3.6.15    
     Build type: release    
    
    Configuration    
           File: /home/hrii/.taskrc (found), 1467 bytes, mode 100644    
           Data: /home/hrii/Documents/organization/tasks_testing (found), dir, mode 40755    
        Locking: Enabled    
             GC: Enabled    
        $EDITOR: nano    
         Server:     
             CA: -    
    Certificate: -    
            Key: -    
          Trust: strict    
        Ciphers: NORMAL    
          Creds:     
    
    Hooks    
         System: Enabled    
       Location: /home/hrii/Documents/organization/tasks_testing/hooks    
                 (-none-)    
    
    Tests    
       Terminal: 165x107    
           Dups: Scanned 2 tasks for duplicate UUIDs:    
                 No duplicates found    
     Broken ref: Scanned 2 tasks for broken references:    
                 No broken references found

**Comment by tbabej** [comment]
Hi @Nlar,

thanks for pointing this out! I think this is likely a case of docs being a little bit misleading. While the default `next` report is ordering tasks by urgency, reports in general can sort based on any attribute. I think this section in the documentation is talking about ordering of the tasks in the report: 

```
task show | grep report | grep sort
report.active.sort                  project+,start+
report.all.sort                     entry-
report.blocked.sort                 due+,priority-,start-,project+
report.blocking.sort                urgency-,due+,entry+
report.completed.sort               end+
report.list.sort                    start-,due+,project+,urgency-
report.long.sort                    modified-
report.ls.sort                      start-,description+
report.minimal.sort                 project+/,description+
report.newest.sort                  entry-
report.next.sort                    urgency-
report.oldest.sort                  entry+
report.overdue.sort                 urgency-,due+
report.ready.sort                   start-,urgency-
report.recurring.sort               due+,urgency-,entry+
report.unblocked.sort               due+,priority-,start-,project+
report.waiting.sort                 due+,wait+,entry+
```

you can see that by default `blocked` and `unblocked` reports are the only ones that consider priority: in those reports ordering of the values of the priority UDA matters.

**Comment by Nlar** [comment]
Thanks for explaining that.  I'll do a pull request for the docs.
